### DISCUSSION TITLE
Find files containing the searched expression

### DISCUSSION THREAD

**Discussion by AtomicNess123** [BurntSushi/ripgrep#1599] [General] [answered]
Is it possible to find the files (and not each hit) of the searched expression? I would just need to know which files contain the term, without having a line for each result. Thanks!

**Comment by BurntSushi** [comment, marked as answer]
Yup. Use `-l/--files-with-matches` flag. Alternatively, the `-c/--count` flag will emit the same output, but with the total number of matches in each file.

**Reply by AtomicNess123** [reply]
Thanks!
